templates/ovn-dashboard.jsonnet

```diff
@@ -180,6 +180,18 @@ local ovnCNIDel = genericGraphLegendPanel('CNI Request DEL Latency', 's').addTar
   )
 );
 
+local ovnLatencyCalculate = genericGraphLegendPanel('Duration for OVN to apply network configuration','s').addTarget(
+  prometheus.target(
+    'histogram_quantile(0.99, sum(rate(ovnkube_master_network_programming_duration_seconds_bucket[1m])) by (pod, le))',
```

Comment:
You should a period of 2m in rate, rather than 1m

Reply:
Curious question, why 2m in rate?

Reply:
Some context -> https://www.robustperception.io/what-range-should-i-use-with-rate/

Reply:
Done, updated!